docs/Caching.md

```diff
@@ -2,7 +2,7 @@
 cBioPortal provides the option of caching information on the backend to improve performance. Without caching, every time a request is received by the backend, a query is sent to the database system for information, and the returned data is processed to construct a response. This may lead to performance issues as the entire process can be rather costly, especially for queries on larger studies. With caching turned on, query responses can be taken directly from the cache if they have already been constructed. They would only be constructed for the initial query.
 
 ## Cache Configuration
-The portal is configured to use Ehcache for backend caching; caching configuration is specified inside an xml file under `persistence/persistence-api/src/main/resources`. The default configuration is heap-only; however, EHCache supports both a disk-only and hybrid (disk + heap) mode. Refer to comments in `ehcache.xml` to switch between cache configurations. The configuration files also specify which caches to create. Additional specifications such as cache size and location are set inside `portal.properties` (more information [here](portal.properties-Reference.md#ehcache-settings)).
+The portal is configured to use Ehcache for backend caching; caching configuration is specified inside an xml file under `persistence/persistence-api/src/main/resources`. The default configuration is a hybrid (disk + heap); however, EHCache supports both a disk-only and heap-only mode. Refer to comments in `ehcache.xml` to switch between cache configurations. The configuration files also specify which caches to create. Additional specifications such as cache size and location are set inside `portal.properties` (more information [here](portal.properties-Reference.md#ehcache-settings)).
```

Comment:
Can we make the casing of "EHCache" consistent?

Reply:
The case pattern used by the project (at least here: https://github.com/ehcache/ehcache3) is "Ehcache", so yes - I think we should change EHCache to Ehcache here.